engine/runtime/src/test/java/org/enso/interpreter/test/MetaIsATest.java

```diff
@@ -95,11 +95,46 @@ public void checkTextsAreText() {
     }
   }
 
+  private void assertType(String msg, Value value, Value... types) {
+    var error = new StringBuilder();
+    for (var type : types) {
+      var r = isACheck.execute(value, type);
+      if (r.asBoolean()) {
+        return;
```

Comment:
Is this short circuit here on purpose? Shouldn't we go through all ty `isACheck.execute` for all the types?